lms/djangoapps/courseware/testutils.py

```diff
@@ -81,7 +81,7 @@ def login(self):
         """
         Logs in the test user.
         """
-        self.client.login(username=self.user.username, password='test')
+        self.client.login(username=self.user.username, password='Password1234')
```

Comment:
Why not `TEST_PASSWORD`?

Reply:
`TEST_PASSWORD` was not available from the superclass for this test class so I just did a manual update.